src/hardware/serialport/softmodem.cpp

```diff
@@ -57,7 +57,8 @@ static const char phoneValidChars[] = "01234567890*=,;#+>";
 static bool MODEM_IsPhoneValid(const std::string &input) {
 	size_t found = input.find_first_not_of(phoneValidChars);
 	if (found != std::string::npos) {
-		LOG_MSG("SERIAL: Phone %s contains invalid character %c", input.c_str(), input[found]);
+		LOG_MSG("SERIAL: Phonebook %s contains invalid character %c.",
```

Comment:
This file is almost entirely 1:1 type changes and LOG_MSG cleanup.
I've flagged the one area of actual change.